frontend/src/app/components/resizer/main-menu-resizer.component.ts

```diff
@@ -26,12 +26,11 @@
 // See docs/COPYRIGHT.rdoc for more details.
 //++
 
-import {ChangeDetectorRef, Component, ElementRef, OnDestroy, OnInit} from '@angular/core';
+import {ChangeDetectorRef, Component, ElementRef, OnInit} from '@angular/core';
 import {distinctUntilChanged} from 'rxjs/operators';
-import {Subscription} from 'rxjs';
-import {untilComponentDestroyed} from 'ng2-rx-componentdestroyed';
 import {ResizeDelta} from "core-app/modules/common/resizer/resizer.component";
 import {MainMenuToggleService} from "core-components/main-menu/main-menu-toggle.service";
+import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
 
 export const mainMenuResizerSelector = 'main-menu-resizer';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/85e9c35a9c9361e0038637188a645cba25065d78/frontend/src/app/components/resizer/main-menu-resizer.component.ts#L36"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>